src/views/pages/items.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ListenerComponent from '../../lib/listener-component.jsx';
+import ItemStore from '../../store/item-store.js';
+import { viewModel } from '../../lib/api-client.js'
+import { getModelsForNewItem } from '../../lib/api-client';
+import Item from '../components/item.jsx';
+export default class Models extends ListenerComponent {
+
+    getStores() {
+        return [
+            ItemStore
+        ];
+    }
+    getState() {
+        return {
+            items: ItemStore.getAllItems()
+        };
+    }
+    newItem(e) {
+        getModelsForNewItem();
+    }
+    render() {
+        return (
+            <div id="item">
+                <h1>All Items</h1>
+                <button onClick={this.newItem.bind(this)}>Make new Item</button>
+                {this.state.items.map(((item, key) => {
+                    return <div key={key}>
+                        <Item item={item} />
+                    </div>
+                }))}
```

Comment:
Is there an extra unnecessary parenthesis-pair? I think there should be parens around that JSX.